workflows/snvphylnfc.nf

```diff
@@ -173,7 +173,7 @@ workflow SNVPHYL {
     // Concat filtered densities to make new invalid_postions
     invalid_positions = CONSOLIDATE_BCFS.out.filtered_densities
         .concat(FIND_REPEATS.out.repeats_bed_file)
-        .collect().map{it -> [[id: 'invalid_positions'], it]}
+        .collect().map{it -> [[id: 'cat_invalid_positions'], it]}
```

Comment:
This was changed because there was a small problem with CAT_CAT where if the only file being cat'd was `invalid_positions.bed` and the output file (the id) was called `invalid_positions`, then it error'd because they had the same name. The name doesn't matter much though, as it is an intermediate file.